pkg/container/container.go

```diff
@@ -62,6 +73,25 @@ type Container interface {
 	// SelinuxLabel returns the container's SelinuxLabel
 	// it takes the sandbox's label, which it falls back upon
 	SelinuxLabel(string) ([]string, error)
+
+	// spec functions
+
+	// returns the spec
+	Spec() *generate.Generator
+
+	// SpecAddMount adds a mount to the container's spec
+	// it takes the rspec mount object
+	// if there is already a mount at the path specified, it removes it.
+	SpecAddMount(rspec.Mount)
+
+	// SpecAddAnnotations adds annotations to the spec.
+	SpecAddAnnotations(sandbox *sandbox.Sandbox, containerVolume []oci.ContainerVolume, mountPoint, configStopSignal string, imageResult *storage.ImageResult, isSystemd, systemdHasCollectMode bool) error
+
+	// SpecConfigureUlimits handles the ulimits of the spec
+	// if an annotation with a path to a ulimit spec has been passed in,
+	// it replaces the default ulimits with the profile
+	// it takes the default ulimit profile
```

Comment:
this line is not completely clear - should it be `otherwise it takes the default ulimit profile`

Reply:
fixed!